Automattic/wp-calypso
# Offer Reset: create a filter bar for the new Plans page

#### Changes proposed in this Pull Request

* Create a filter bar that includes a product type filter (Security, Performance, All) and a duration (monthly, annually) filter.

#### Note

* This doesn't include the logic to make the bar sticky. It might be better to do that in another PR once we have added the Cart bar (the one in the top of this one) to the page because both bars should be sticky.

#### Testing instructions

* Run this PR.
* Activate the `offerResetFlow` AB test.
* Visit `http://calypso.localhost:3000/plans/:site`.
* Check both selectors work and they look as per design.

Fixes 1169247016322522-as-1187323761633614

#### Demo
![image](https://user-images.githubusercontent.com/3418513/89447907-2d326900-d72d-11ea-8530-e20cb9279de4.png)
![image](https://user-images.githubusercontent.com/3418513/89447949-3f140c00-d72d-11ea-8566-d10f35964b7f.png)
![Kapture 2020-08-05 at 15 07 27](https://user-images.githubusercontent.com/3418513/89448076-6e2a7d80-d72d-11ea-9ea8-8e92b9b3fdf6.gif)
